Makefile

```diff
@@ -1,8 +1,23 @@
-.PHONY: install-hooks
+.PHONY: install-hooks build check
+
+ifneq (${DAEMON},)
+GRADLE_ARGS := --daemon
+endif
+
+GRADLE := ${CURDIR}/gradlew ${GRADLE_ARGS}
 
 venv:
 	virtualenv venv
-	./venv/bin/pip install pre-commit
+	${CURDIR}/venv/bin/pip install pre-commit
+
+.git/hooks/pre-commit: venv
+	${CURDIR}/venv/bin/pre-commit install --install-hooks
+
+install-hooks: ${CURDIR}/.git/hooks/pre-commit
+	@true
+
+build:
+	${GRADLE} plugin:build
 
-install-hooks: venv
-	./venv/bin/pre-commit install --install-hooks
+generateSwagger assembleDebug check:
+	${GRADLE} $@
```

Comment:
Can we clean this up a bit?
This https://github.com/gradle/gradle-completion looks like a great alternative ;)

Reply:
Definitely correct.
As you noticed the makefile targets were mainly created to have tab completion. 
I'm going to remove the commit